instance/tests/js/instance_app_spec.js

```diff
@@ -37,6 +37,7 @@ describe('Instance app', function () {
     });
 
     describe('Index controller', function() {
+        /* TEMPORARILY comment out the Jasmine tests - to be corrected in next commit or PR.
```

Comment:
@bradenmacdonald Makes sense - thanks for the details. And kudos for pulling this to the finish line - OpenCraft IM just got a whole lot better : )